std/src/main/scala/algebra/std/boolean.scala

```diff
@@ -9,6 +10,13 @@ package object boolean extends BooleanInstances
 trait BooleanInstances {
   implicit val booleanAlgebra: BooleanAlgebra =
     new BooleanAlgebra
+
+  val BooleanRing = new BoolRing[Boolean] {
```

Comment:
up to you. If so, we could do `Field[Boolean] with BoolRing[Boolean]`. I'm not sure I'd bother in this PR, personally.